stripe/src/main/java/com/stripe/android/model/Source.java

```diff
@@ -298,6 +322,11 @@ public void setSourceTypeData(Map<String, Object> sourceTypeData) {
         mSourceTypeData = sourceTypeData;
     }
 
+    public void setTypeRaw(@NonNull @Size(min = 1) String typeRaw) {
+        mTypeRaw = typeRaw;
+        setType(UNKNOWN);
```

Comment:
I'm not sure I want to totally abandon setting the non-raw ("cooked?") type as UNKNOWN when the user overrides the actual type, since I have some logic based on what enum-type things are. Let me take a look and see if it's safe to do away with.